artifacts/rhdh-plugins-reference/argocd/argocd-plugin-user.adoc

```diff
@@ -27,8 +27,9 @@ image::rhdh-plugins-reference/sidebar.png[Sidebar]
 +
 image::rhdh-plugins-reference/deployment_summary.png[Deployment summary]
 
-
+////
 [role="_additional-resources"]
 .Additional resources
 
 * For more information on dynamic plugins, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/configuring_plugins_in_red_hat_developer_hub/rhdh-installing-dynamic-plugins[Dynamic plugin installation].
+////
```

Comment:
The link is  still visible in https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-591/plugin-rhdh/#_using-the-argo-cd-plugin